scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -797,3 +796,58 @@ def update_motr_hare_keys_for_all_nodes(self):
                     f" {host}:{self._motr_hare_conf}")
             execute_command(self, cmd)
 
+def lvm_clean(self):
+    self.logger.info("Removing cortx lvms")
+    vol_grps=execute_command(self, "vgs|grep vg_srvnode|awk '{print $1}'")[0].split('\n')[0:-1]
+    if (len(vol_grps) == 0):
+        self.logger.info("No cortx volume groups (e.g. vg_srvnode-1_md1) are found \n")
+        return
+    self.logger.info("Executing swapoff -a")
+    swap_off(self)
+    self.logger.info(f"Removing cortx LVM entries from {FSTAB}")
+    execute_command(self, f"sed -i.bak '/vg_srvnode/d' {FSTAB}")
```

Comment:
instead of reading the volume groups from sed command, we have to use the hare_conf keys and remove them

Reply:
Done
72ad34457bd132ba2e62b04120d54cc0d416f0d3